components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -168,19 +168,18 @@ func (r *Reconciler) reconcileNATSBackend(ctx context.Context, backendStatus *ev
 	if err != nil {
 		backendStatus.SetPublisherReadyCondition(false, eventingv1alpha1.ConditionReasonPublisherProxySecretError, err.Error())
 		if updateErr := r.syncBackendStatus(ctx, backendStatus, nil); updateErr != nil {
-			return ctrl.Result{}, errors.Wrapf(err, "update status when delete publisher proxy secret failed")
+			return ctrl.Result{}, errors.Wrapf(err, "failed to update status while deleting Event Publisher secret")
 		}
-		return ctrl.Result{}, errors.Wrapf(err, "delete eventing publisher proxy secret failed")
+		return ctrl.Result{}, errors.Wrapf(err, "delete eventing Event Publisher secret failed")
```

Comment:
```
failed to delete....
``` 
??

Reply:
I thought about it, but the thing is: the `errors.Wrapf` is essentially the error message, so according to the golang standards we shouldn't capitalise the error output. And the ticket was also only about the logging. I only touched the errors for consistency with the logs (like components naming) and for better sentence flow. But in general the error messages were not part of that PR.